### ISSUE TITLE
Integration of dialogue system with robot

### ISSUE THREAD

**Issue by AmeBel** [opencog/opencog#1204]
Substituting the OpenCog-based dialogue system in place of the OpenDial-based system for the OpenCog-based robotics framework. Tweaking the various speech act schema and associated hard-coded dialogue rules for appropriateness in a robotics context 

Subtle issues here would seem to be:
- Goals specific to the game world context, and their connection with speech acts
- Articulation of descriptions of things in the game world environment
- Articulation of events, processes and sequences in the game world

**Comment by linas** [comment]
FYI, I have started trying to do this here: https://github.com/opencog/ros-behavior-scripting  I pretty much haven't gotten anywhere yet; just laying down scaffolding right now.  Its more of a proof-of-concept.

**Comment by anitzkin** [comment]
As you're thinking about this . . . it occurred to me yesterday as I was brainstorming about the dialogue system -- 
you will want to include a stage in the formation of the action scripts that synchronizes the gestures ( as in all non-verbal
behavior) with the verbal behavior such that gestures are timed to co-occur with (or immediately before or after?) stressed
words, phrase endings, etc.  Obviously some fun experimentation required here!

--------------------------------------------
On Thu, 5/7/15, Linas Vepštas <notifications@github.com> wrote:

 Subject: Re: [opencog] Integration of dialogue system with robotics framework (#1204)
 To: "opencog/opencog" <opencog@noreply.github.com>
 Date: Thursday, May 7, 2015, 8:45 AM
 
 FYI, I have started trying to do
 this here: https://github.com/opencog/ros-behavior-scripting
  I pretty much haven't gotten anywhere yet; just laying
 down scaffolding right now.  Its more of a
 proof-of-concept.
 
 —
 Reply to this email directly or view
 it on GitHub.

**Comment by linas** [comment]
Yes, eventually. Right now, its just laying down bare-bones, and fixing crashes. Trying to lay down enough of a framework that perhaps others can join in eventually.  Right now, nothing works yet.

**Comment by linas** [comment]
I am working on this now ...

**Comment by linas** [comment]
Milestone reached with integration of chatbot with openpsi in the last few months...
